release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -0,0 +1,1534 @@
+to upgrade the data model, an embedded etcd configuration option in no longer
+allowed. Embedded is not co-located and mainly used in single-master
+deployments. Migration scripts will convert the v3 data model and allow you to
+move an embedded etcd to an external etcd either on the same host or a different
+host than the masters. In addition, there is a new scale up ability for etcd
+clusters.
+
+See
+xref:../install_config/upgrading/migrating_embedded_etcd.adoc#install-config-upgrading-etcd-data-migration[Migrating
+Embedded etcd to External etcd] for more information.
+
+[[ocp-37-modular-installer]]
+==== Modular Installer to Allow Playbooks to Run Independently
+
+The installer has been enhanced to allow administrators to install specific
+components. By breaking up the roles and playbooks, there is better targeting of
+ad hoc administration tasks.
+
+[[new-install-experience-around-phases]]
+==== New Installation Experience Around Phases
+When you run the installer, {product-title} now reports back at the end what
+phases you have gone through.
+
+If the installation fails during a phase, you will be notified on the screen
+along with the errors from the Ansible run. Once you resolve the issue, rather
+than run the entire installation over again, you can pick up from the failed
+phase. This results in an increased level of control during installations and
+results in time savings.
+
+[[ocp-37-increased-control-over-image-stream-templates]]
+==== Increased Control Over Image Stream and Templates
+With {product-title} 3.7, there is added control over whether or not your cluster
+automatically upgrades all the content provided during cluster upgrades.
+
+Edit the `openshift_install_examples` variable in the hosted file or set it as a variable in the installer.
+
+----
+RPM = /etc/origin/examples /etc/origin/hosted
+Container = /usr/share/openshift/examples /usr/share/openshift/hosted
+
+openshift_install_examples=false
+----
+
+Setting `openshift_install_examples` to `false` will cause the installer to not
+upgrade the imagestream and templates. `True` is the default behavior.
+
+[[ocp-37-install-config-cfme-from-ocp-installer]]
+==== Installation and Configuration of CFME 4.6 from the OpenShift Installer
+
+CloudForms Management Engine (CFME) 4.6 is now fully supported running on
+{product-title} 3.7 as a set of containers. CFME is an available API endpoint on
+all {product-title} clusters that choose to use it. More cluster administrators
+are now able to leverage CFME and begin experiencing the insight and automations
+available to them in {product-title}.
+
+To install CFME 4.6:
+
+----
+# ansible-playbook -v -i <YOUR_INVENTORY>
+playbooks/byo/openshift-management/config.yml
+----
+
+To configure CFME 4.6 to consume the {product-title} installation it is running on:
+
+----
+# ansible-playbook -v -i <YOUR_INVENTORY>
+playbooks/byo/openshift-management/add_container_provider.yml
```

Comment:
This playbook still has a known issue: https://bugzilla.redhat.com/show_bug.cgi?id=1506951